docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -324,22 +380,58 @@ The table below maps the Bitbucket Cloud and Bitbucket Server roles to the corre
+
+To review the additional permissions granted by the Organization Manager role, see the tables for each Git provider:
+
+-   [GitHub](#permissions-for-github)
+-   [GitLab](#permissions-for-gitlab)
+-   [Bitbucket](#permissions-for-bitbucket)
+
+!!! important
+    Organization managers can access the **Policies** and **Integrations** settings sections of your organization and can therefore impact some repository settings for all repositories of your organization, even repositories that they can't access on the Git provider. However, they can't access the repositories themselves and can only see the repository names.
+
+### Assigning the Organization Manager role
+
+To assign the Organization Manager role:
+
+1.  Open your organization **Settings**, page **Roles and permissions**.
+
+1.  In the **Organization managers** area, use the search field to find the relevant organization member and click the member's name.
+
+    !!! note
+        You can only assign the Organization Manager role to [members of your organization](./managing-people.md#joining).
+
+    ![Security and risk management access management](images/roles-permissions-organization-manager-assign.png)
+
+### Revoking the Organization Manager role
+
+To revoke the Organization Manager role:
+
+1.  Open your organization **Settings**, page **Roles and permissions**.
+
+1.  In the **Organization managers** area, scroll the list to find the relevant user.
+
+1.  Click the **Revoke role icon** to the right of the user's name and confirm.
+
 ## Configuring who can change the analysis configuration {: id="change-analysis-configuration"}
 
-By default, only users with the Codacy permission level **Repository Write** can change analysis configurations.
+By default, only users with the Codacy role **repository write** can change analysis configurations.
 
-To change this, open your organization **Settings**, page **Member privileges**, and define the lowest Codacy permission level required to perform the following operations on the repositories of your organization:
+To change this, open your organization **Settings**, page **Roles and permissions**, and define the lowest Codacy role required to perform the following operations on the repositories of your organization:
 
 -   [Ignore issues](../repositories/issues.md#ignoring-and-managing-issues)
 -   [Ignore files](../repositories-configure/ignoring-files.md)
 -   [Configure code patterns](../repositories-configure/configuring-code-patterns.md)
 -   [Configure file extensions](../repositories-configure/file-extensions.md)
 -   [Manage branches](../repositories-configure/managing-branches.md)
 
-![Configuring who can change analysis configurations](images/organization-analysis-configuration.png)
+![Configuring who can change analysis configurations](images/roles-permissions-repo-management.png)
```

Comment:
Are the screenshots updated? 🤔 
We are renaming Codacy **permission levels** by **roles**, right? However, the UI is not consistent with this change, it still mentions **the lowest permission level**... Will the UI change at a later phase?

Reply:
Yes, these UI updates are scheduled to happen after the launch ([TAROT-2431](https://codacy.atlassian.net/browse/TAROT-2431)). I'll update this then; the UI and docs may be inconsistent for a little while.